core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1380,6 +1421,13 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCK_WRITER_THREAD_QUEUE_WAITING_TASK_COUNT =
+      new Builder("Worker.BlockWriterThreadQueueWaitingTaskCount")
+          .setDescription("The current number of tasks waiting in the work queue "
+              + "in the block write thread pool")
```

Comment:
```suggestion
              + "in the block writer thread pool")
```
In this is bounded by a property key, pls include in the description as above